javascript/src/main/App.js

```diff
@@ -71,12 +74,14 @@ function App() {
           <AuthorizedRoute path="/tutorAssignments/new" exact component={NewTutorAssignment} authorizedRoles={["admin", "member"]} />
           <AuthorizedRoute path="/tutorAssignments/edit/:tutorAssignmentId" exact component={EditTutorAssignment} authorizedRoles={["admin", "member"]} />
           <AuthorizedRoute path="/admin/setActiveQuarter" exact component={QuarterFilter} authorizedRoles={["admin"]} />
+          <Route path="/courses/show/:courseId" component={CourseShow} />
+          <Route path="/courses/show/:id" component={CourseList} />
```

Comment:
I noticed that the paths for these two routes are identical, and from testing the app, it looks like only the first one is being used. Is the second route being used, and if not, can we remove it?